packages/client-web-kit-theme/src/styles/list.css

```
@@ -21,10 +21,10 @@
 }
 
 .list .list-item {
-    background-color: #ececec;
-    border: 1px solid #dedede;
+    background-color: var(--vc-color-bg-muted);
+    border: 1px solid var(--vc-color-border);
 
-    box-shadow: 0 4px 25px 0 rgba(0,0,0,.1);
+    box-shadow: 0 4px 25px 0 rgba(0, 0, 0, .1);
```

Stylelint (17.11.1)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)